src/status_im2/contexts/quo_preview/profile/profile_card/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.quo-preview.profile.profile-card.view
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im.react-native.resources :as resources]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.contexts.quo-preview.profile.profile-card.profile-card :as profile-card]))
```

Comment:
this will then just be imported as [quo2.core :as quo] and then use profile card as `quo/profile-card`

Reply:
I answered it in https://github.com/status-im/status-mobile/pull/14652#discussion_r1060137675
Please I would do the changes needed, I have no problem with doing changes but you guys should have a definition about this because I did it like than, and I changed it because it was requested, and now i have to change it again.

Reply:
Thanks @mattgle, there is a clear definition already in place. Any component that is in the Figma components file should be in the quo2 library. It seems like there is just some confusion we need to address. Apologies for the inconvenience and your work is very much appreciated! :) 

Please tag me in the original conversation so I can see the discussion and see why this should be an exception to the rule we have in place.